src/control/cmd/dmg/pretty/storage_test.go

```diff
@@ -291,7 +311,12 @@ host[0-1023] 1 B (1 module) 1 B (1 controller)
 			if err := PrintResponseErrors(resp, &bld); err != nil {
 				t.Fatal(err)
 			}
-			if err := PrintHostStorageMap(resp.HostStorage, &bld); err != nil {
+			if tc.displaySpaceUsage {
```

Comment:
Probably should split this into two different tests, since this is testing two different functions.